small_ha_cluster.md

```diff
@@ -222,7 +222,7 @@ Read [this tutorial](http://www.tutorialspoint.com/hadoop/hadoop_multi_node_clus
 - Change the `/usr/local/elasticsearch/config/elasticsearch.yml` file as shown below. This is minimal and allows all hosts to act as backup masters in case the acting master goes down. Also all hosts are data/index nodes so can respond to queries and host shards of the index.
 
 ```
-cluster.name: your-app-name
+cluster.name: some-cluster # write the same cluster name as in elasticsearch section on pio-env.sh
 discovery.zen.ping.multicast.enabled: false # most cloud services don't allow multicast
```

Comment:
as I said on the other PR the app-name is not quite what you want here, a cluster descriptor that defines the Elasticsearch machines is better.